code/modules/mob/living/simple_animal/friendly/farm_animals.dm

```diff
@@ -233,6 +233,8 @@
 				C.name = name
 			if(mind)
 				mind.transfer_to(C)
+			if(pcollar)
+				C.pcollar = pcollar
```

Comment:
This just sets the variable, but doesn't change the name of the chicken or keep the death alarm of it
```suggestion
			if(pcollar)
				var/the_collar = pcollar
				unEquip(pcollar)
				C.add_collar(the_collar )
```

Reply:
Watch the space on that suggested change as well